app/plugins/logging.js

```diff
@@ -5,6 +5,6 @@ module.exports = {
   options: {
     logPayload: true,
     prettyPrint: config.isDev,
-    level: config.isDev ? 'debug' : 'warn'
+    level: 'warn'
```

Comment:
Reducing logging level to be consistent with other Hapi services as `debug` is overkill.